### ISSUE TITLE
Change Device in GUI / Link Device to Profile

### ISSUE THREAD

**Issue by dmadison** [psieg/Lightpack#153]
This is a feature request: it would be nice to be able to change the current device in the GUI without having to manually edit `main.conf` or re-run the configuration wizard. Better yet, devices could be linked to profiles so that a profile change would also change devices.

I know this is rather niche as most people won't have more than one ambilight attached to their PC, but at the very least it's useful for switching to/from a virtual device for debugging.

**Comment by MatteoN74** [comment]
Indeed. And maybe also an indicator of which ports are active? (like in the Arduino IDE menu)

**Comment by zomfg** [comment]
Same. Right now I never know what stuff is in the profile and what is in the main config, and when I make new profiles for testing different situations it "breaks" the previous ones. Moving more settings to profiles would be handy indeed.

**Comment by Hukuma1** [comment]
+1. I actually had to search for the `main.conf` as the profile you can edit within the GUI doesn't provide the ability to edit the COM port. Would love to add at least the ability to edit COM port right into the Settings -> Device tab. Even a simple drop down box just to switch between different ports would be welcome.
